core/camel-core-model/src/main/java/org/apache/camel/model/rest/VerbDefinition.java

```diff
@@ -129,6 +132,15 @@ public String getLabel() {
         }
     }
 
+    @Override
+    public void addOutput(ProcessorDefinition<?> processorDefinition) {
+        if (route == null) {
+            route = new RouteDefinition();
+        }
+
+        route.addOutput(processorDefinition);
+    }
```

Comment:
Not sure if it is correct, I've opened it mainly to track the gaps I'm finding when working on additional DSLs. 

In this specific case I'd need each definition that support output nodes, to be properly identifiable and the reason is that in the YAML DSL I need a way to apply common patterns as example:

```yaml
- rest:
    paths:
      "/foo":
        get:
          type: com.github.lburgazzoli.camel.dsl.yaml.support.model.MyFooBar
          out-type: com.github.lburgazzoli.camel.dsl.yaml.support.model.MyBean
          steps:
            - to: "direct:bar"
            '''.stripLeading()))
```            

The auto generated DSL parser is able to automatically handle the `steps` property, but since the `VerbDefinition` is not market as an `OutputNode`, it does not know how to attach such steps to the definition, hence I've added this method and related interfaces.

Reply:
Yeah looks fine for me, but a good idea is to build camel-spring and see if the XML XSD is changed unexpectedly